### PR TITLE
fix(angular_1_router): Renamed transpiled require functions

### LINKED ISSUE #6914: router(angular1): Router build file not compatible with browserify for bundling
Using browserify for bundling causes errors using the angular_1_router.js file due to browserify trying to import the `require` statements in the built router file. A workaround is to rename all the occurrences of `require('` to `routerRequire('` and change the `var require = function...` to `var routerRequire = function...` in the `angular_1_router.js` file.

### PR DESCRIPTION
**Pull request by brandonroberts**
Closes #6914 

The require function was causing failures when using bundlers such as Browserify and SystemJS.

cc: @petebacondarwin

### CODE REVIEW

**Comment by petebacondarwin** [comment]
LGTM

File: modules/angular1_router/build.js [hunk 1]
```diff
@@ -49,6 +49,7 @@ function main(modulesDirectory) {
 var IMPORT_RE = new RegExp("import \\{?([\\w\\n_, ]+)\\}? from '(.+)';?", 'g');
 var INJECT_RE = new RegExp("@Inject\\(ROUTER_PRIMARY_COMPONENT\\)", 'g');
 var IMJECTABLE_RE = new RegExp("@Injectable\\(\\)", 'g');
```
Comment (ericmartinezr) [inline modules/angular1_router/build.js]: I know this is not part of your PR, but shouldn't that line say `INJECTABLE_RE`?

Comment (petebacondarwin) [inline modules/angular1_router/build.js]: +1

**Comment by brandonroberts** [comment]
Got it @ericmartinezr

**Comment by vikerman** [comment]
@petebacondarwin - Please add pr_state: LGTM if you think this is ready to be merged

**Comment by vikerman** [comment]
TAP Green

**Pull request closed without merging**